src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const MIN_DAYS_FOR_DISCOUNT = 3;
+  const LONG_TERM = 7;
+  const PRICE = 40;
+  let sale = 20;
+
+  if (days >= MIN_DAYS_FOR_DISCOUNT && days < LONG_TERM) {
+    return days * PRICE - sale;
+  }
+
+  if (days >= LONG_TERM) {
+    sale = 50;
+
+    return days * PRICE - sale;
```

Comment:
u can move it to variable, something like - `defaultPrice = days + price` and then decrease this value